### ISSUE TITLE
IRChecker does not verify method existence on receiver

### ISSUE THREAD

**Issue by gzm0** [scala-js/scala-js#1419]
It does check existence for fields.

**Comment by sjrd** [comment]
Now that #3831 is merged, we can delay this to post-v1.0.0.

**Comment by gzm0** [comment]
After #4410, it is IMO clear that this is the Analyzers job (and IIUC it does this already). So unless you object, @sjrd, I'd propose to close this as wontfix.

**Comment by sjrd** [comment]
Sounds good to me. 😉
